back/src/Guide/Types/Core.hs

```diff
@@ -187,15 +188,19 @@ data Item = Item {
   _itemConsDeleted :: [Trait],         -- ^ Deleted cons go here
   _itemEcosystem   :: MarkdownBlock,   -- ^ The ecosystem section
   _itemNotes       :: MarkdownTree,    -- ^ The notes section
-  _itemLink        :: Maybe Url,       -- ^ Link to homepage or something
-  _itemKind        :: ItemKind         -- ^ Is it a library, tool, etc
+  _itemLink        :: Maybe Url        -- ^ Link to homepage or something
   }
   deriving (Show, Generic, Data)
 
-deriveSafeCopySorted 11 'extension ''Item
+deriveSafeCopySorted 12 'extension ''Item
 makeFields ''Item
 
-changelog ''Item (Current 11, Past 10) []
+changelog ''Item (Current 12, Past 11)
+  [Removed "_itemKind"  [t|ItemKind|],
+   Added "_itemHackage" [hs|Nothing|]]
```

Comment:
Migrations shouldn't lose data. This migration loses data – whatever Hackage name was contained in the `_itemKind` will be lost.

You can fetch the data contained in `_itemKind` here, look at https://github.com/aelve/guide/blame/c32852ab771fb116ddd7c4d937a7c9f4377313ea/back/src/Guide/Types/Core.hs#L271 for inspiration.